framework/docker/internal/docker_keyring_test.go

```diff
@@ -35,7 +36,7 @@ func (s *DockerKeyringTestSuite) SetupSuite() {
 	dockerClient, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
 	s.Require().NoError(err)
 
-	s.dockerClient = dockerClient
+	s.dockerClient = tastoraclient.NewClient(dockerClient, "docker-keyring-test-suite")
```

Comment:
Q: is the `"docker-keyring-test-suite"` fixed for all tests? is it safe to do so if two tests are running concurrently on the same machine? will they share their keyring?

Reply:
oh good call, I think we need to move this to SetupTest and add a random element to the name